Externals.cfg

```diff
@@ -1,9 +1,11 @@
 [regional_workflow]
 protocol = git
-repo_url = https://github.com/ufs-community/regional_workflow
+#repo_url = https://github.com/ufs-community/regional_workflow
+repo_url = https://github.com/chan-hoo/regional_workflow
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = d45d2ac
+branch = feature/update_uwm_hash
+#hash = d45d2ac
 local_path = regional_workflow
```

Comment:
This is a temporary change. Once the corresponding PR (#796) in the regional workflow is merged, this will be updated.